Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'gpx'
 gem "activeadmin", git: "https://github.com/activeadmin/activeadmin"
 gem "inherited_resources", git: "https://github.com/activeadmin/inherited_resources"
 gem "activerecord-import"
+gem "delayed_job_active_record"
```

Comment:
https://evilmartians.com/chronicles/open-source-software-whats-in-a-poke - part about background processing.
Delayed_job seems to be a little bit outdated and badly maintained solution. Also, do we really need this new table in DB?)